WcaOnRails/config/routes.rb

```diff
@@ -168,6 +168,7 @@
       get '/persons/:wca_id' => "persons#show", as: :person
       resources :competitions, only: [:index, :show] do
         get '/wcif' => 'competitions#show_wcif'
+        post '/wcif/events' => 'competitions#update_events_from_wcif', as: :update_events_from_wcif
```

Comment:
The other API endpoint is a `PATCH`, right? Any reason you chose to do a `POST` here instead?

Reply:
Certainly not, changed!